boards/DCM/Src/App/torquevectoring/App_TorqueVectoring.c

```diff
@@ -0,0 +1,148 @@
+#include "torquevectoring/App_TorqueVectoring.h"
+#include "torquevectoring/App_TorqueVectoringConstants.h"
+#include "torquevectoring/App_PowerLimiting.h"
+#include "torquevectoring/App_ActiveDifferential.h"
+#include "torquevectoring/App_TractionControl.h"
+#include "torquevectoring/App_Regen.h"
+#include "App_Timer.h"
+#include "App_CanRx.h"
+#include "App_CanTx.h"
+#include "App_SharedMacros.h"
+
+TimerChannel pid_timeout;
+
+PowerLimiting_Inputs       power_limiting_inputs;
+ActiveDifferential_Inputs  active_differential_inputs;
+ActiveDifferential_Outputs active_differential_outputs;
+TractionControl_Inputs     traction_control_inputs;
+TractionControl_Outputs    traction_control_outputs;
+Regen_Inputs               regen_inputs;
+Regen_Outputs              regen_outputs;
+
+PID pid_power_correction;
+// NOTE: Correction factor centered about 0.0f
+float pid_power_correction_factor = 0.0f;
+PID   pid_traction_control;
+
+float accelerator_pedal_percent;
+float wheel_speed_front_left_kph;
+float wheel_speed_front_right_kph;
+float motor_speed_left_rpm;
+float motor_speed_right_rpm;
+float battery_voltage;
+float current_consumption;
+float left_motor_temp_C;
+float right_motor_temp_C;
+float available_battery_power_kW;
+float steering_angle_deg;
```

Comment:
I think these should all be static: https://stackoverflow.com/questions/2271902/static-vs-global